ECHO-COVID19.ipynb

```diff
@@ -403,7 +234,7 @@
    },
```

Comment:
Given that this is now a final report, would it be appropriate to add some analysis after the figures?